File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:

        if (x<0):
            return False
        
        num = x
        rev = 0
      
        while( num != 0):
            l_digit = num%10
            rev = ( rev*10 ) + l_digit
            num = num // 10
        
        if (rev == x):
            return True
        else:
            return False
          

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {

        int num = x;
        int rev = 0;

        
        if(x<0){
            return false;
        }
        while(num!=0){
            int l_digit = num%10;
            rev = (rev*10) + l_digit;
            num = num/10;

        }
        if (x==rev){
            return true;
        }
        else{
            return false;
        }
    }
}